Source/SelfService/Web/microservice/microserviceView/healthStatus/helpers.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React, { useState, useEffect } from 'react';
+
+import { useSnackbar } from 'notistack';
+
+import { getPodLogs } from '../../../api/api';
+
+import { GridRenderCellParams } from '@mui/x-data-grid-pro';
+import { IconButton, Link } from '@mui/material';
+import { Close, DownloadRounded } from '@mui/icons-material';
+import Slide from '@mui/material/Slide';
+
+export const formatTime = (time: string) => {
```

Comment:
Is this only used for rounding the `age` field of a container to seconds? If so, I think it would maybe make more sense to simply round it in the platform-api - something like (in [k8s_repo.go](https://github.com/dolittle/platform-api/blob/main/pkg/platform/k8s/k8s_repo.go#L259)):
```golang
age := time.Since(pod.CreationTimestamp.Time).Round(time.Second)
```
@jakhog ?